File: core-java-practice/gcr-codebase/KilometerToMiles.java
import java.util.Scanner;
public class KilometerToMiles {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // Input distance in kilometers
        System.out.print("Enter distance in kilometers: ");
        double kilometers = sc.nextDouble();
        // Conversion formula
        double miles = kilometers * 0.621371;
        // Output result
        System.out.println("Distance in miles = " + miles);

    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/KilometerToMiles.cs

class KilometerToMiles {
    public static void Main(String[] args) {
      
        // Input distance in kilometers
        Console.Write("Enter distance in kilometers: ");
        double kilometers= ConvertToDouble(Console.ReadLine());

        //Conversion Formula
        double miles=kilometers * 0.621371;

        //Output 
        Console.WriteLine("Distance in miles: " + miles);
    
    }
}
